### PR TITLE
SCP bug fix: SCP does not copy file exceeds 2,147,483,648 bytes. (https://github.com/PowerShell/Win32-OpenSSH/issues/145)

### PR DESCRIPTION
**Pull request by bingbing8**
1. update the install-sshd does not failed when log folder exists.
2. enable to copy files larger than 2G

### CODE REVIEW

File: contrib/win32/win32compat/w32fd.c [hunk 1]
```diff
@@ -814,7 +814,10 @@ int
 w32_ftruncate(int fd, off_t length) {
     CHECK_FD(fd);
 
-    if (!SetFilePointer(w32_fd_to_handle(fd), length, 0, FILE_BEGIN))
+	LARGE_INTEGER new_postion;
```
Comment (manojampalam) [inline contrib/win32/win32compat/w32fd.c]: Move declarations to top of code block (C style, to be consistent with rest of OpenSSH code)

**Review by manojampalam — APPROVED** [review]

File: contrib/win32/win32compat/w32fd.c (hunk 1 above)
Comment (bingbing8) [reply contrib/win32/win32compat/w32fd.c]: fixed!

File: contrib/win32/win32compat/w32fd.c [hunk 2]
```diff
@@ -814,7 +814,10 @@ int
...
+	memset(&new_postion, 0, sizeof(new_postion));
```
Comment (remittor) [inline contrib/win32/win32compat/w32fd.c]: and why is it necessary?

Comment (bingbing8) [reply contrib/win32/win32compat/w32fd.c]: not necessary. removed it.

File: contrib/win32/win32compat/w32fd.c [hunk 3]
```diff
@@ -812,19 +812,22 @@ int w32_allocate_fd_for_handle(HANDLE h, BOOL is_sock) {
...
+	LARGE_INTEGER new_postion;
+
+	CHECK_FD(fd);
 
-    if (!SetFilePointer(w32_fd_to_handle(fd), length, 0, FILE_BEGIN))
-        return -1;
-    if (!SetEndOfFile(w32_fd_to_handle(fd)))
-        return -1;
+	new_postion.QuadPart = length;
+	if (!SetFilePointerEx(w32_fd_to_handle(fd), new_postion, 0, FILE_BEGIN))
```
Comment (bagajjal) [inline contrib/win32/win32compat/w32fd.c:820]: curious to understand why it is working for sftp.

also check the sftp put/get with the new changes.

**Pull request merged**